RMtools_1D/do_QUfit_1D_mnest.py

```diff
@@ -231,7 +231,7 @@ def load_model(modelNum, verbose=False):
             spec.loader.exec_module(mod)
         except:
             print(
-                "Model could not be found! Please make sure model is present either in {}/models_ns/, or in {}/RMtools_1D/models_ns/".format(
```

Comment:
Is this really a bug? `models_ns` is is only in the 1D module... Although I'm not sure how RMtools_dir is defined